src/types/dateTime.ts

```diff
@@ -0,0 +1,12 @@
+import {StrongType, makeStrong} from '../strong-type';
+
```

Comment:
Filename should match the project styles:

- [ ] Change from `dateTime.ts` to `date-time.ts`